docs/content/getting_started/create_an_app.md

```diff
@@ -0,0 +1,46 @@
+# Create an Appplication
+Creating your very own MOOSE application is a cinch: you do it by "Forking the Stork!"... which is our application template:
+
+### Choose a new application name.
+* Consider the use of an acryonym. We prefer animal names for applications, but you are free to choose whatever name suits your needs.
+
+### Fork Stork!
+* Go to the Stork GitHub Page and Fork the Stork! by clicking the "Fork" button in the upper right of the page.
+
+### Rename your fork
+* Click on your repository, then the Settings button
+* Rename your repository
+##### IMPORTANT: Rename your repository on Github to match your animal name!
```

Comment:
This should be [Admonitions](https://pythonhosted.org/Markdown/extensions/admonition.html) only "note", "warning" and "danger" are available, we might need to add more.